README.md

```diff
@@ -255,91 +252,6 @@ Options:
 
 ```
 
-#### Check
-
-##### `--check`
-
-Does not write any formatted code back to file. It will instead check whether any
-changes *would* be made. It returns one of three possible exit codes:
-
-**0** - indicates **no changes** would be made
-
-```
-$ echo 'include: "foo.txt"' | snakefmt --check -                                        
-[INFO] 1 file(s) would be left unchanged 🎉
-$ echo "Exit code: $?"
-Exit code: 0
-```
-
-**1** - indicates **changes** would be made
-
-```
-$ echo 'include:"foo.txt"' | snakefmt --check - 
-[INFO] 1 file(s) would be changed 😬
-$ echo "Exit code: $?"
-Exit code: 1
-```
-
-**123** - indicates there was an **error** such as invalid syntax
-
-```
-$ echo 'include:' | snakefmt --check -            
-[ERROR] L2: In include definition.
-[INFO] 1 file(s) contains errors 🤕
-$ echo "Exit code: $?"
-Exit code: 123
-```
-
-#### Compact diff
-
-##### `--compact-diff`
-
-Does not write any formatted code back to file. It will instead print a compact diff of
-how the code looks before and after formatting. The diff is compact as it only prints
-the lines that will change, with a few lines of surrounding context.
-
-```
-$ echo 'x = 1\ny = 3\n\n\nrule foo:\n\tinput: "foo.txt"' | snakefmt --compact-diff -
-=====> Diff for stdin <=====
-
---- original
-+++ new
-@@ -3,4 +3,5 @@
- 
- 
- rule foo:
--       input: "foo.txt"
-+    input:
-+        "foo.txt",
-
-[INFO] All done 🎉
-```
-
-The above example shows that the variable assignments at the beginning of the file are
-not included in the compact diff (but would be included in a full diff).
-
-#### Diff
-
-##### `--diff`
-
-Does not write any formatted code back to file. It will instead print a diff of how the
-code looks before and after formatting.
-
-```
-$ echo 'rule foo:\n\tinput: "foo.txt"' | snakefmt --diff -
-=====> Diff for stdin <=====
-
-  rule foo:
--       input: "foo.txt"
-+     input:
-+         "foo.txt",
-
-[INFO] All done 🎉
-```
-
-If multiple files are specified, a diff for each file is written to stdout, separated by
-`=====> Diff for <filepath> <=====`.
-
```

Comment:
I disagree with deleting this

Reply:
why? as i've said, the options already appear in the README in the Full Usage section, and appear at CLI when running `--help`. and even just the keyword itself,  check/diff, is pretty self-explanatory.

Reply:
nm